typescript/nomad-deploy/src/bridge/index.ts

```diff
@@ -350,6 +353,104 @@ export async function enrollBridgeRouter(
   );
 }
 
+/**
+ * Deploy any preconfigured custom tokens. These tokens should be specified by
+ * ID in the configuration block
+ *
+ * @param local - The deploy instance for the chain on which to deploy custom
+ *  tokens
+ */
+export async function deployCustoms(local: AnyBridgeDeploy): Promise<void> {
+  local.contracts.customs = [];
```

Comment:
Perhaps just add check here that `local.contracts.customs == undefined` so nobody misuses `deployCustoms` post deploy time and doesn't accidentally overwrite existing custom reprs

Reply:
dffd972